app/views/directives/traffic-table.html

```diff
@@ -14,9 +14,9 @@
   <tbody ng-if="(portsByRoute | hashSize) == 0">
     <tr><td colspan="7"><em>{{emptyMessage || 'No routes or ports to show'}}</em></td></tr>
   </tbody>
-  <tbody ng-if="(portsByRoute | hashSize) > 0">
-    <tr ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tr>
-    <tr ng-repeat="port in ports" ng-if="routeName !== ''">
+  <tbody ng-if="(portsByRoute | hashSize) > 0" ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tbody>
+  <tbody ng-repeat="port in ports" ng-if="routeName !== ''">
```

Comment:
A solution would be to add the `2px` top border to `tbody > tr + tr` instead. That way, we could keep the `ng-repeat` in the `tr` element, and update the `pods-table` directive and `routes.html` to do this as well.

cc @jwforres